index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/b9edfb7622.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Apple</title>
+</head>
+
+<body>
+  <div class="container">
+    <div class="landing-section">
+      <nav class="nav-link-items">
+        <ul>
+          <li><a href="#"><i class="fab fa-apple"></i></a></li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Mac</a></li>
+          <li><a href="#">iPod</a></li>
+          <li><a href="#">iPhone</a></li>
+          <li><a href="#">iPad</a></li>
+          <li><a href="#">iTunes</a></li>
+          <li><a href="#">Support</a></li>
+        </ul>
+        <form class="search-nav"><input type="search" id="search-input"></form>
+      </nav>
+      <div class="welcome-section">
+        <a href="#" class="question size"> What will your verse be</a>
+        <a href="#" class="share size">Everyone has something to share. See how people are using iPad in their unique
+          ways to add to the world's story</a>
+        <a href="#" class="verse size"> <span class="text-content-link">Watch the "Your Verse" film and explore the
+            stories</span>
+          <span><i class="far fa-caret-square-right"></i></span></a>
+      </div>
+    </div>
+    <div class="apple-products">
+      <div class="product"><img src="./images/i5.jpg" alt="iphon product" class="fluid-img"></div>
+      <div class="product"><img src="./images/i5c.jpg" alt="iphon product" class="fluid-img"></div>
+      <div class="product"><img src="./images/birthday.jpg" alt="iphon product" class="fluid-img"></div>
+      <div class="product"><img src="./images/supply.jpg" alt="iphon product" class="fluid-img"></div>
+    </div>
```

Comment:
## [REQUIRED] - CHANGE - Elements to links.
- [ ] These four elements should be links, let's add them inside of the 'a' tags.